tools/custom-checks/bump-openapi-spec-dep-version.ts

```diff
@@ -0,0 +1,161 @@
+import fs from "fs-extra";
+import path from "path";
+import { fileURLToPath } from "url";
+import { globby, Options as GlobbyOptions } from "globby";
+import { RuntimeError } from "run-time-error";
+import { isStdLibRecord } from "./is-std-lib-record";
+import { exec } from "child_process";
+import util from "util";
+
+interface paths {
+  post: {
+    requestBody: unknown;
+  };
+}
+interface packageJson {
```

Comment:
@aldousalvarez Please follow naming conventions and have the interface start with I and also be CapitalCased in general.